src/main/kotlin/io/inventi/eventstore/eventhandler/IdempotentEventHandler.kt

```diff
@@ -112,7 +125,14 @@ abstract class IdempotentEventHandler(
                     method.invoke(this@IdempotentEventHandler, eventData)
 
                 } catch (e: Exception) {
-                    logger.error("Failure on method invocation. eventRecord: $event streamName: $streamName groupName: $groupName")
+                    logger.error("""
+                       Failure on method invocation ${method.name}:
+                       eventId: ${event.eventId}
+                       eventType: ${event.eventType}
+                       eventData: ${String(event.data)}
```

Comment:
`new String(bytes[])` is locale dependant. Please specify `UTF8` as encoding

Reply:
This is actually kotlin's String function: 
```public inline fun String(bytes: ByteArray): String =
    java.lang.String(bytes, Charsets.UTF_8) as String```